src/Components/Publishing/ToolTip/ArtistToolTip.tsx

```diff
@@ -120,6 +122,10 @@ export const ArtistContainer = styled.div`
       color: black;
     }
   }
+  ${MarketDataSummaryContainer} {
+    ${unica("s12")};
```

Comment:
It's a bit annoying that we have to make the size overwrite here. Wondering if there is a way to pass a prop - something like `text size - xsm, sm or medium` and based on that define style in the MarketData component?